### DISCUSSION TITLE
Fail to load gpt4chan_model_16

### DISCUSSION THREAD

**Discussion by Patrick-81** [oobabooga/text-generation-webui#3790] [General]
My PC runs Linux Mint with 32Gb of memory and a RTX 3060 12Gb graphic card and I can't load this damn model. The UI answered that:

torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 32.00 MiB (GPU 0; 11.75 GiB total capacity; 10.71 GiB already allocated; 10.38 MiB free; 10.72 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation. See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

Is there any way to fix it ?
Thanks

**Comment by missionfloyd** [comment]
1. Follow the instructions [here](https://github.com/oobabooga/text-generation-webui#gpt-4chan).
2. Move the contents of `models/EleutherAI_gpt-j-6B` to the gpt 4chan folder. 
3. Load it with Transformers and check `load-in-8bit` or `load-in-4bit`.
